### PR TITLE
Fixes #5: Add details to addons, plugins, gfx (pair=@n1k0)

### LINKED ISSUE #5: Add support for more content-related properties to records
Quoting Jorge:

> String name
> String why
> String who 
> String bug
> These properties are used to display the block on AMO: https://addons.mozilla.org/firefox/blocked/i990. In the future, I'd like to at least merge "why" and "who", but ideally I'd prefer to have a set of predefined and localized explanations, with the option to have custom text, since most blocks have the same copy.

### PR DESCRIPTION
**Pull request by magopian**

### CODE REVIEW

File: amo-blocklist.json [hunk 1]
```diff
@@ -149,11 +149,32 @@
                 }
               }
             }
+          },
+          "details": {
```
Comment (Natim) [inline amo-blocklist.json]: Then we should change that in xml2kinto too :+1:

**Comment by Natim** [comment]
r+

File: amo-blocklist.json (hunk 1 above)
Comment (magopian) [inline amo-blocklist.json]: Filed https://github.com/mozilla-services/xml2kinto/issues/35

File: amo-blocklist.json [hunk 2]
```diff
@@ -286,8 +307,24 @@
               "GREATER_THAN",
               "GREATER_THAN_OR_EQUAL"
             ]
+          },
```
Comment (Natim) [inline amo-blocklist.json]: We do not do any scrapping for GFX, should we remove that?

**Pull request merged**